src/features/moderation/logs/ModlogItem.tsx

```diff
@@ -99,24 +156,67 @@ function renderModlogData(item: ModlogItemType): LogEntryData {
 
 export function ModlogItem({ item }: ModLogItemProps) {
   const buildGeneralBrowseLink = useBuildGeneralBrowseLink();
-  const { title, by, when, message, link } = renderModlogData(item);
+  const {
+    icon,
+    title,
+    when,
+    by,
+    role: role_,
+    message,
+    reason,
+    expires,
+    link,
+  } = renderModlogData(item);
+
+  const isAdmin = useIsAdmin(
+    "admin" in item
+      ? item.admin
+      : "moderator" in item
+        ? item.moderator
+        : undefined,
+  );
+
+  const role =
+    by && isAdmin
+      ? "admin-local"
+      : "admin" in item
+        ? "admin-remote"
+        : role_ ?? "mod";
 
   return (
     <IonItem
       className={maxWidthCss}
+      href={undefined}
       routerLink={link ? buildGeneralBrowseLink(link) : undefined}
       detail={false}
     >
-      <Contents>
-        <TitleLine>
-          <Title>{title}</Title>
-          <aside>
-            {by && <span>{by} · </span>}
-            <Ago date={when} />
-          </aside>
-        </TitleLine>
-        <Message>{message}</Message>
-      </Contents>
+      <Container>
+        <StartContent>
+          <IonIcon icon={icon} color="medium" />
```

Comment:
I see you're using the styles from InboxItem, which is great! Do you think its worth abstracting them into a component that takes `icon`? Perhaps for another task?

Reply:
Good question. I did indeed copy some of the styled components from there in order to have the design stay consistent across the app, thinking it would make sense to have the modlog look similar to a view that users will already be familiar with, that way it would be easier to read.

The reason I didn't extract them into shared components is because they are still separate features and I didn't want to unecessarily entangle them in case they evolve somewhat differently in the future.